dts/bindings/serial/nxp,lpc11u6x-uart.yaml

```diff
@@ -14,12 +14,12 @@ properties:
     clocks:
       required: true
 
-    pinmuxs:
-      type: phandle-array
+    pinctrl-0:
+      type: phandles
       required: true
-      description: RX and TX pinmux
+      description: Pinmux selection for RX and TX pins
 
-    pinmux-names:
+    pinctrl-names:
       type: string-array
       required: true
-      description: UART pins names
+      description: Pin control names
```

Comment:
no need to re-define type/description for `pinctrl-` fields, they're defined in `pinctrl-device.yaml` (check other cases as well).

Reply:
Dropped these re-definitions where they were present